src/domain/usecases/program-rules-processing/ProgramRuleValidationForEGASP.ts

```diff
@@ -153,7 +153,6 @@ export class ProgramRuleValidationForEGASP {
         return {
             type: "event",
             eventId: event.event || "",
-            teiId: event.trackedEntityInstance,
```

Comment:
Check if it is necessary and if so add in d2-api in the interface D2TrackerEventBase the property `trackedEntity` (before it was `trackedEntityInstance`)

Reply:
You're right, seems like it's needed on the trackedEntity like before, I've added it. Thanks!